src/IssueProcessor.ts

```diff
@@ -374,7 +376,12 @@ export class IssueProcessor {
       event => event.event === 'labeled' && event.label.name === label
     );
 
-    return staleLabeledEvent!.created_at;
+    if (!staleLabeledEvent) {
```

Comment:
@hross FYI

Reply:
awesome. thanks for fixing my bad code :)